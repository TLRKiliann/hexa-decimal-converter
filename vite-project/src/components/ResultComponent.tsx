type ResultBinProps = {
  result: number | string | undefined;
}

export default function ResultComponent(props: ResultBinProps) {
  return (
    <>
      <span data-testid="span-testid">
        Result : {props.result ? props.result : "Not binary !!!"}
      </span>
    </>
  )
}
