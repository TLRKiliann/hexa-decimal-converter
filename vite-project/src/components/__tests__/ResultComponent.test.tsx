import { test, expect } from "vitest"
import { create } from "react-test-renderer"
import ResultComponent from '../ResultComponent';

test('Snapshat of ResultComponent', () => {
    const tree = create(<ResultComponent result={undefined} />)
    expect(tree.toJSON()).toMatchSnapshot()
});