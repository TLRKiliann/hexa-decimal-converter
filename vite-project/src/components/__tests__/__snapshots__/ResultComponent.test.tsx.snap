// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshat of ResultComponent 1`] = `
<span
  data-testid="span-testid"
>
  Result : 
  Not binary !!!
</span>
`;
